eth/vm/computation.py

```diff
@@ -568,14 +577,19 @@ def apply_computation(cls,
         """
         with cls(state, message, transaction_context) as computation:
             # Early exit on pre-compiles
-            if message.code_address in computation.precompiles:
-                computation.precompiles[message.code_address](computation)
+            precompile = computation.precompiles.get(message.code_address, NO_RESULT)
+            if precompile is not NO_RESULT:
+                precompile(computation)
                 return computation
 
             show_debug2 = computation.logger.show_debug2
 
+            opcode_lookup = computation.opcodes
             for opcode in computation.code:
-                opcode_fn = computation.get_opcode_fn(opcode)
+                try:
+                    opcode_fn = opcode_lookup[opcode]
```

Comment:
Wondering if you checked if this is faster/slower `opcode_lookup.get(opcode, INVALID)`?  Where `INVALID` uses the `computation.code_stream` to lookup what the `opcode` is at the current code position?

Not needed for this pr, but just stood out to me as *maybe* another similar styled speedup.

Reply:
Hm, yeah I don't remember exactly, but it sounds like an alternative I tried and rejected. I think `get` had more overhead that `__getitem__` and the exception rarely happens, plus the `try` is cheap to run in the happy case. I'll see if I can find anything in a quick re-run.

Reply:
Tried a couple variations without getting any speedup